src/libs/AxelarQueryAPI.ts

```diff
@@ -187,17 +189,28 @@ export class AxelarQueryAPI {
 
     const { gas_price } = sourceToken;
 
-    const destTxFee = parseEther(gas_price).mul(gasLimit);
+    const gasPriceEther = parseEther(gas_price);
 
-    if (gasMultiplier > 1) {
-      return destTxFee
-        .add(baseFee)
-        .mul(gasMultiplier * 10000)
-        .div(10000)
-        .toString();
+    let _gasPrice;
+
+    if (minGasPrice) {
+      _gasPrice = gasPriceEther.sub(BigNumber.from(minGasPrice)).gt(0)
+        ? gasPriceEther
+        : BigNumber.from(minGasPrice);
+    } else {
+      _gasPrice = gasPriceEther;
     }
```

Comment:
nice. this looks correct. btw, can you simplify the logic here a bit? For example,
- gasPriceEther.sub(BigNumber.from(minGasPrice)).gt(0) -> gasPriceEther.gt(minGasPrice) (this function also accepts a string, it will automatically convert string to BigNumber internally)
- we set the default value for `minGasPrice` to "0", so we can remove the else block.

Reply:
```ts
let _gasPrice = parseEther(gas_price)
_gasPrice = _gasPrice.gt(minGasPrice) ? _gasPrice : BigNumber.from(minGasPrice);
```